### PR TITLE
docs: add instructions for accessing the Swagger UI

### PR DESCRIPTION
**Pull request by waqaraqeel**
Signed-off-by: Waqar Aqeel <waqeel@fb.com>

## Summary

Add subsection in [Quick Start Guide](https://magma.github.io/magma/docs/next/basics/quick_start_guide) to access the Swagger UI on local cloud.

### CODE REVIEW

**Review by electronjoe — APPROVED** [review]

File: docs/readmes/basics/quick_start_guide.md [hunk 1]
```diff
@@ -150,6 +150,14 @@ installed client certificates. See [here](https://support.globalsign.com/custome
 for instructions. If you use Chrome or Safari, you may have to restart the
 browser before the certificate can be used.
 
+### Accessing the Swagger UI
+After starting the Orchestrator with `HOST [magma/orc8r/cloud/docker]$ ./run.py`
+and importing `admin_operator.pfx`, you should be able to visit the Swagger UI
+at [https://localhost:9443/swagger/v1/ui](https://localhost:9443/swagger/v1/ui).
+Note that your browser may refuse to accept the server certificate from
+`localhost:9443`. Firefox and Safari will let you override this warning, but
+Chrome will not.
```
Comment (hcgatewood) [inline docs/readmes/basics/quick_start_guide.md]: Merge your contribution with the previous section -- doesn't need to be a separate h3

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Thanks for adding! One small request then we can merge

File: docs/readmes/basics/quick_start_guide.md (hunk 1 above)
Comment (waqaraqeel) [reply docs/readmes/basics/quick_start_guide.md]: Good point! Thanks! Done.

**Review by hcgatewood — APPROVED** [review]
Thanks for the addition!

**Pull request merged**